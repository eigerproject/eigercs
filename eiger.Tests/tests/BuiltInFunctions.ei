~ emitln
emitln("Hello, World!")

~ double
emitln(double("3.1415"))

~ int
emitln(int("3.1415"))
emitln(int(3.1415))

emitln(int("3.8"))
emitln(int(3.8))