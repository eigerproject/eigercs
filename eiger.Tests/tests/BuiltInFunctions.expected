Hello, World!
3.1415
3
3
4
4